pyproject.toml

```
@@ -5,6 +5,24 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "crypt_r"
 version = "3.12.3"
+maintainers = [
+    {name = "Miro Hrončok", email = "miro@hroncok.cz"},
+    {name = "Petr Viktorin", email = "encukou@gmail.com"},
+    {name = "Tomáš Hrnčiar", email = "tomas.hrnciar@me.com"},
+    {name = "Karolina Surma", email = "ksurma@redhat.com"},
+]
+description = "A copy of the `crypt` module that was removed in Python 3.13"
```

I am unsure how PyPI will respond to the backticks, so let's be careful and skip them maybe:

```suggestion
description = "A copy of the crypt module that was removed in Python 3.13"
```

---

I took inspiration from zombie-imp, see: https://pypi.org/project/zombie-imp/